compiler/rustc_span/src/hygiene.rs

```diff
@@ -1326,12 +1326,10 @@ pub fn decode_syntax_context<D: Decoder, F: FnOnce(&mut D, u32) -> SyntaxContext
 
     let outer_ctxts = &context.remapped_ctxts;
 
-    // Ensure that the lock() temporary is dropped early
-    {
+    // the lock() temporary is dropped early
         if let Some(ctxt) = outer_ctxts.lock().get(raw_id as usize).copied().flatten() {
             return ctxt;
         }
-    }
```

Comment:
I don't get why the braces were there in the first place, the `lock` is dropped after the `if` anyway